src/SoftPosit.jl

```diff
@@ -1,10 +1,5 @@
 module SoftPosit
 
-    # import SoftPosit_jll
```

Comment:
BTW, you can probably remove https://github.com/milankl/SoftPosit.jl/blob/fd3f61b673fea5bcf628a0dee00961d43be95d3b/Project.toml#L6 from the dependencies, right?

Reply:
Yes, we should do that.